src/ui/wxWidgets/MenuEditHandlers.cpp

```diff
@@ -370,6 +371,21 @@ void PasswordSafeFrame::OnClearClipboardClick(wxCommandEvent& WXUNUSED(evt))
 
 void PasswordSafeFrame::OnCopyPasswordClick(wxCommandEvent& evt)
 {
+  if (PWSprefs::GetInstance()->GetPref(PWSprefs::DontAskQuestion)) {
+    wxRichMessageDialog dialog(this, 
+      _("Pressing OK will copy the password of the selected item\nto the clipboard. The clipboard will be securely cleared\nwhen Password Safe is closed.\n\nPressing Cancel stops the password being copied."), 
```

Comment:
```suggestion
      _("Pressing OK will copy the password of the selected item\nito the clipboard. The clipboard will be securely cleared\nwhen Password Safe is closed.\n\nPressing Cancel stops the password from being copied."), 
```

Reply:
Thank you, I also adapted the text of the Windows version, from where I took the text.